tests/test_json_io.py

```diff
@@ -279,7 +273,8 @@ def test_plugin_event_json(self):
         self.assertEqual(getattr(getattr(event_loaded, "ev"), "_session_id"), "EV-001")
 
     def test_unplug_event_json(self):
-        _ = self._obj_compare_helper(self.unplug_event)
+        event_loaded = self._obj_compare_helper(self.unplug_event)
+        self.assertEqual(getattr(getattr(event_loaded, "ev"), "_session_id"), "EV-001")
```

Comment:
@zach401 this fixes the serialization issue I was referring to.

Reply:
Great!